core/src/main/java/org/infinispan/persistence/file/SingleFileStore.java

```diff
@@ -141,13 +149,17 @@ public void start() {
    public void stop() {
       try {
          if (channel != null) {
-            log.tracef("Stopping store %s, size = %d, file size = %d", ctx.getCache().getName(), entries.size(), channel.size());
+            log.tracef("Stopping store %s, size = %d, file size = %d", ctx.getCache().getName(), size(), channel.size());
 
             // reset state
             channel.close();
             channel = null;
```

Comment:
Technically it should be possible to get a NPE from the channel being null in the load method (or see it in a closed state).